src/main/steps/respondent/support-you-need-during-case/documents-support/content.ts

```diff
@@ -40,25 +40,25 @@ const en = {
 };
 
 const cy: typeof en = {
-  section: 'Reasonable adjustments',
-  title: 'I need documents in an alternative format',
+  section: 'Addasiadau rhesymol',
+  title: 'Rwyf angen dogfennau mewn fformat amgen',
   courtCommunication:
-    'Think about all communications with the court, as well as what you might need at a hearing. Consider remote and in-person hearings, in case your preferred hearing type is not possible.',
-  optionHint: 'Select all that apply to you',
-  summaryText: 'Contacts for help',
-  docsColour: 'Documents in a specified colour',
+    'Meddyliwch am yr holl ohebiaeth â’r llys, ynghyd â’r hyn y gallwch fod ei angen mewn gwrandawiad. Ystyriwch wrandawiadau o bell a rhai wyneb yn wyneb, rhag ofn bod y math o wrandawiad o’ch dewis ddim yn bosibl.',
+  optionHint: "Dewiswch bob un sy'n berthnasol i chi",
+  summaryText: 'Cysylltiadau am gymorth',
+  docsColour: 'Dogfennau mewn lliw penodol',
   docsColourDetails: 'Describe what you need',
-  docsReadFormat: 'Documents in an easy read format',
+  docsReadFormat: 'Dogfennau mewn fformat hawdd i’w darllen',
   docsReadFormatHint: 'information written in simple language with pictures',
-  brailleDocs: 'Braille documents',
-  largePrintDocs: 'Documents in large print',
+  brailleDocs: 'Dogfennau Braille',
+  largePrintDocs: 'Dogfennau mewn print bras',
   largePrintDocsDetails: 'Describe what you need',
   audioTranslation: 'Audio translation of documents',
```

Comment:
@altairhm These are also available, please incorporate it.
audioTranslation: 'Audio translation of documents',
docsReadFormatHint: 'information written in simple language with pictures',

Reply:
@altairhm Please incorporate them.